dissect/target/plugins/general/osinfo.py

```diff
@@ -0,0 +1,36 @@
+from typing import Callable, Generator
+
+from flow.record import Record
+
+from dissect.target import plugin
+from dissect.target.exceptions import PluginError
+from dissect.target.helpers.record import TargetRecordDescriptor
+
+OSInfoRecord = TargetRecordDescriptor(
+    "generic/osinfo",
+    [
+        ("string", "prop"),
```

Comment:
```suggestion
        ("string", "name"),
```
Perhaps more clear than `prop`? Or `key`.